android/src/main/java/com/revenuecat/purchases/hybridcommon/mappers/StoreProductMapper.kt

```diff
@@ -0,0 +1,73 @@
+package com.revenuecat.purchases.hybridcommon.mappers
+
+import com.revenuecat.purchases.models.StoreProduct
+
+fun StoreProduct.map(): Map<String, Any?> =
+    mapOf(
+        "identifier" to sku,
+        "description" to description,
+        "title" to title,
+        "price" to priceAmountMicros / 1000000.0,
```

Comment:
I think this was here before but could we have this be `1_000_000.0` for readability?